package.json

```diff
@@ -29,14 +29,14 @@
     "test": "pnpm lint && vitest run --coverage"
   },
   "devDependencies": {
-    "@vitest/coverage-c8": "^0.31.4",
-    "changelogen": "^0.5.3",
-    "eslint": "^8.42.0",
-    "eslint-config-unjs": "^0.2.0",
-    "prettier": "^2.8.8",
-    "typescript": "^4.9.5",
+    "@vitest/coverage-v8": "^0.34.1",
```

Comment:
Replace `@vitest/coverage-c8` with `@vitest/coverage-v8`, because the former has been deprecated on npm.

Reply:
And this is why CI on #80 fails.